archive.php

```diff
@@ -5,7 +5,6 @@
 <div class="container">
```

Comment:
Consider adding `mb-5` to this container to increase the amount of space between the main content and footer, particularly on smaller screen sizes

Reply:
Done